provider/terraform/virtual_fields.rb

```diff
@@ -23,6 +23,16 @@ class Terraform < Provider::AbstractCore
     # behaviour. They often don't map to underlying API fields (although they
```

Comment:
```suggestion
    # behaviour. They don't map to underlying API fields (although they
```

& the although... bit